dev/com.ibm.ws.jaxrs.2.0_fat/test-applications/paramconverter/src/com/ibm/ws/jaxrs/fat/paramconverter/objects/NoPublicConstructorListObject.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxrs.fat.paramconverter.objects;
+
+import java.util.ArrayList;
+
+import javax.json.bind.annotation.JsonbCreator;
+
+public class NoPublicConstructorListObject extends ArrayList<String> {
+
+    private static final long serialVersionUID = 646543532165146L;
+
+    @JsonbCreator
```

Comment:
Does this work?  When we chatted, I thought you said that `@JsonbCreator` only worked on other (non-no-arg) constructors.  If so, we should remove it here - or change the static getter to be a public constructor with an argument.

Reply:
It didn't work and I missed taking this out. I adjusted the test to return a string instead and it works.